test/unit/serverConfigParams.test.ts

```diff
@@ -0,0 +1,215 @@
+        });
+        expect(() => verifyServerConfiguration()).to.throw(
+          '"blockchain.workerMinBalance" failed custom validation because Invalid BigNumber string representation: 1e10'
+        );
+      });
+
+      it('Should pass if the string representation of a BigNumber is correct', function () {
+        config.util.extendDeep(config, {
+          blockchain: {
+            workerMinBalance: '1035386848354684654698546564654654',
+          },
+        });
+        expect(() => verifyServerConfiguration()).not.to.throw();
+      });
+
+      it('Should fail if an address is incorrect', function () {
+        const wrongAddress = Wallet.createRandom()
+          .address.toString()
+          .slice(0, -1);
+
+        config.util.extendDeep(config, {
+          app: { sponsoredDestinations: [wrongAddress] },
+        });
+        expect(() => verifyServerConfiguration()).to.throw(
+          'Server configuration error:  "app.sponsoredDestinations[0]" failed custom validation because Invalid address:'
+        );
+      });
+
+      it('Should pass if an address is correct', function () {
+        const correctAddress = Wallet.createRandom().address.toString();
+
+        config.util.extendDeep(config, {
+          app: { sponsoredDestinations: [correctAddress] },
+        });
+        expect(() => verifyServerConfiguration()).not.to.throw();
+      });
+
+      it('Should fail if a PK is incorrect', function () {
+        const wrongPK = Wallet.createRandom()
+          .privateKey.toString()
+          .slice(0, -1);
+
+        config.util.extendDeep(config, {
+          register: { privateKey: wrongPK },
+        });
+        expect(() => verifyServerConfiguration()).to.throw(
+          'Server configuration error:  "register.privateKey" failed custom validation because Invalid PK:'
+        );
+      });
+
+      it('Should pass if a PK is correct', function () {
+        const correctPK = Wallet.createRandom().privateKey.toString();
+
+        config.util.extendDeep(config, {
+          register: { privateKey: correctPK },
+        });
+        expect(() => verifyServerConfiguration()).not.to.throw();
+      });
+
+      it('Should fail if the mnemonic is incorrect', function () {
+        const wrongMnemonic = Wallet.createRandom().mnemonic.phrase.slice(
+          0,
+          -1
+        );
+
+        config.util.extendDeep(config, {
+          register: { mnemonic: wrongMnemonic },
+        });
+        expect(() => verifyServerConfiguration()).to.throw(
+          'Server configuration error:  "register.mnemonic" failed custom validation because Invalid Mnemonic:'
+        );
+      });
+
+      it('Should pass if the mnemonic is correct', function () {
+        const correctMnemonic = Wallet.createRandom().mnemonic.phrase;
+
+        config.util.extendDeep(config, {
+          register: { mnemonic: correctMnemonic },
+        });
+        expect(() => verifyServerConfiguration()).not.to.throw();
```

Comment:
can you leave a space between each expect.

Reply:
Done